### PR TITLE
easymod mgn12

### PR DESCRIPTION
**Pull request by EduardoOliveira**
## Part name
Easymod mgn12
## Compatible EVA version
 2.1.0 - 2.2.0
## Related EVA parts
Easymod XY joinners
## Necessary Hardware BOM
3mm longer 5mm screws if you intend to use mgn12 on Y as well
## Description
It enables the usage of mgn12 in the X axis and mgn12c or mgn12h in Y
## Onshape or STEP files
https://cad.onshape.com/documents/8559b0e7f5d3b30427c227c5/w/d70f098c5b327e06a1ae23f4/e/12a2436a47701eb37945bb45?configuration=List_ByYG67Ay22XcMs%3DMGN12 

https://cad.onshape.com/documents/8559b0e7f5d3b30427c227c5/w/d70f098c5b327e06a1ae23f4/e/67166bbf807ca01f4a1ac8b3?configuration=List_V5TmcWwH5u18OA%3DDefault
## Print instructions
To print the adapter plates increase tops and bottoms and infill at ~50%
## Contributor
Me EOliveira @ discord

### CODE REVIEW

File: docs/easymod/em_mgn12.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Easymod mgn12
+---
+
+> Created by: [EOliveira](https://github.com/EduardoOliveira)
+
+An implementation of easymod 1.2.1 using mgn12
+
+The XY joiners are drop in replacement for the official ones.
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:9]: Change `are drop in` to `are a drop in`.

File: docs/easymod/em_mgn12.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This will allow you to use your setup with mgn12 in the X axis, mainly redugin the weight of it.
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:11]: Change `redugin` to `reducing`

File: docs/easymod/em_mgn12.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Easymod mgn12
+---
+
+> Created by: [EOliveira](https://github.com/EduardoOliveira)
+
+An implementation of easymod 1.2.1 using mgn12
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:7]: Change to `# An implementation of MGN12 for EasyMod 1.2.1`

File: docs/easymod/em_mgn12.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+If you are also using mgn12C or MGN12H on the Y axis you'll need the following adapters in place of the original aluminum plates.
+
+You'll need longer M5 screws as this adapters are 3mm thicker than the aluminum part. I recomend printing with more tops/bottoms and higher infill to incease strength.
+
+![xyjoinners](assets/adapters.png)
+
+[Onshape](https://cad.onshape.com/documents/8559b0e7f5d3b30427c227c5/w/d70f098c5b327e06a1ae23f4/e/67166bbf807ca01f4a1ac8b3?configuration=List_V5TmcWwH5u18OA%3DDefault).
+
+
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:26]: add 
```
# Compatible EVA versions
1.2.1
# BOM
```

File: docs/easymod/em_mgn12.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+The XY joiners are drop in replacement for the official ones.
+
+This will allow you to use your setup with mgn12 in the X axis, mainly redugin the weight of it.
+
+[Onshape](https://cad.onshape.com/documents/8559b0e7f5d3b30427c227c5/w/d70f098c5b327e06a1ae23f4/e/12a2436a47701eb37945bb45?configuration=List_ByYG67Ay22XcMs%3DMGN12)
+
+![xyjoinners](assets/xyjoinners.png)
+
+
+If you are also using mgn12C or MGN12H on the Y axis you'll need the following adapters in place of the original aluminum plates.
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:18]: change `mgn12C` to `MGN12C`

File: docs/easymod/em_mgn12.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+You'll need longer M5 screws as this adapters are 3mm thicker than the aluminum part. I recomend printing with more tops/bottoms and higher infill to incease strength.
```
Comment (miklschmidt) [inline docs/easymod/em_mgn12.md:20]: change `as this adapters` to `as these adapters` and `I recomend` to `I recommend`.

**Review by miklschmidt — CHANGES_REQUESTED** [review]
Thank you for your contribution! I have a few change requests before merging, mainly spelling errors and a couple of headers. Other than that, very nice!

**Comment by EduardoOliveira** [comment]
Thank you for the changes

**Review by miklschmidt — APPROVED** [review]

**Pull request merged**